standard/info/wikis/marvelrivals/info.lua

```diff
@@ -0,0 +1,40 @@
+---
+-- @Liquipedia
+-- wiki=marvelrivals
+-- page=Module:Info
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+return {
+	startYear = 2024,
+	wikiName = 'marvelrivals',
+	name = 'Marvel Rivals',
+	defaultGame = 'Marvel Rivals',
+	games = {
+		['Marvel Rivals'] = {
+			abbreviation = 'Marvel Rivals',
+			name = 'Marvel Rivals',
+			link = 'Marvel Rivals',
+			logo = {
+				darkMode = 'Marvel Rivals darkmode.png',
+				lightMode = 'Marvel Rivals lightmode.png',
+			},
+			defaultTeamLogo = {
+				darkMode = 'Marvel Rivals darkmode.png',
+				lightMode = 'Marvel Rivals lightmode.png',
+			},
+		},
+	},
+	config = {
+		squads = {
+			hasPosition = true,
```

Comment:
Is this correct?

Reply:
> Is this correct?

The game has designated in-game position, like Overwatch so I keep this to be set as true.